cli.py

```diff
@@ -27,12 +30,15 @@
 
 from d3a_interface.exceptions import D3AException
 from d3a_interface.utils import iterate_over_all_modules
+from d3a_interface.api_simulation_config import ApiSimulationConfigException
+import d3a_api_client
 
 import d3a_api_client.setups as setups
 from d3a_api_client.constants import SETUP_FILE_PATH, DEFAULT_DOMAIN_NAME, DEFAULT_WEBSOCKET_DOMAIN
 
 
 log = getLogger(__name__)
+api_client_path = os.path.dirname(inspect.getsourcefile(d3a_api_client))
```

Comment:
It's the first time I see this strategy used to retrieve a path. I would have usually done something like:
```python
import os
current_dir = os.path.dirname(os.path.abspath(__file__))  # Directory of the cli.py script
api_client_path = os.path.join(current_dir, 'd3a_api_client')
```
I have no real reason to ask for a change as of now, just wanted to mention it in case you find it useful for the future.

Reply:
Thats something I took as a [reference](https://github.com/gridsingularity/d3a/blob/master/src/d3a/d3a_core/util.py#L42) from `d3a` repo.